tests/distributions/test_custom.py

```diff
@@ -362,32 +362,40 @@ def scan_step(left, right):
             return x, x_update
 
         def dist(size):
-            xs, updates = scan(
-                fn=scan_step,
-                sequences=[
-                    pt.as_tensor_variable(np.array([-4, -3])),
-                    pt.as_tensor_variable(np.array([-2, -1])),
-                ],
-                name="xs",
-            )
+            with pytest.warns(DeprecationWarning, match="Scan return signature will change"):
+                xs, updates = scan(
+                    fn=scan_step,
+                    sequences=[
+                        pt.as_tensor_variable(np.array([-4, -3])),
+                        pt.as_tensor_variable(np.array([-2, -1])),
+                    ],
+                    name="xs",
+                    # There's a bug in the ordering of outputs when there's a mapped `None` output
```

Comment:
Bug is https://github.com/pymc-devs/pytensor/issues/1796

Hopefully we can fix in the next release and update the remaining tests to the new API